rules_common.mk

```diff
@@ -97,18 +98,19 @@ LIB_SRCS += drivers/sonar_i2cxl.cpp
 LIB_SRCS += drivers/sonar_telemetry.cpp
 LIB_SRCS += drivers/spektrum_satellite.cpp
 LIB_SRCS += drivers/state_display.cpp
+LIB_SRCS += drivers/state_display_megafly_rev4.cpp
+LIB_SRCS += drivers/state_display_sparky_v2.cpp
 
-LIB_SRCS += hal/common/dbg.cpp
+# LIB_SRCS += hal/common/dbg.cpp # -> causes link error with chibios
```

Comment:
This is a replacement for `print_util_dbg_print`  that allows nice things like
`dout << "This is the " << n << "th error message" << endl;` 
@basil-huber did it a long time ago but we never converted the code to use it.